http/websockets.go

```diff
@@ -87,6 +104,10 @@ func command(c *fb.Context, w http.ResponseWriter, r *http.Request) (int, error)
 	path = filepath.Clean(path)
 	buff := new(bytes.Buffer)
 
+	if len(commandShell) != 0 {
+		command = append(commandShell, strings.Join(command, " "))
```

Comment:
If `bash -c` or `sh -c` are used, the original command must be passed as a single argument. I.e., command `ls -la ./*` should be executed as `bash -c "ls -la ./*"`. Else, it is executed as `bash -c ls`. That's why I added `strings.Join`.